### PR TITLE
BAPL-853 (for upstream master): Updated descriptions of attributes for guided decision tables

### PR DESCRIPTION
**Pull request by sterobin**
The updated text in this PR comes from a new DRL rules doc, which has been SME, QE, and peer reviewed. Impacted area of UI is GDT designer > Columns > Insert Column > Add an attribute. Descriptions appear to the right. 

See [JIRA](https://issues.jboss.org/browse/BAPL-853) for details.

### CODE REVIEW

File: drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties [hunk 1]
```diff
@@ -54,35 +54,35 @@ AdditionalInfoPageView.UpdateEngine=Update engine with changes:
...
+AttributeColumnPageView.LockOnActiveDescription=A Boolean value, applicable only to rules within rule flow groups or agenda groups. When the option is selected, the next time the ruleflow group for the rule becomes active or the agenda group for the rule receives a focus, the rule cannot be activated again until the ruleflow group is no longer active or the agenda group loses the focus. This is a stronger version of the `no-loop` attribute, because the activation of a matching rule is discarded regardless of the origin of the update (not only by the rule itself). This attribute is ideal for calculation rules where you have a number of rules that modify a fact and you do not want any rule re-matching and firing again.
 AttributeColumnPageView.Negate=negate
-AttributeColumnPageView.NegateDescription=Negate attribute.
+AttributeColumnPageView.NegateDescription=A Boolean value. When the option is selected, the rule is negated.
```
Comment (sterobin) [inline drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties]: @manstis or anyone else, could you please validate my "Negate" description? It's the only one not on my existing list and I redefined it on the fly as I did it. Want to make sure it's accurate.

Comment (jomarko) [reply drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/resources/org/drools/workbench/screens/guided/dtable/client/resources/i18n/GuidedDecisionTableErraiConstants.properties]: The condition/when part of the rule is negated. having this attribute column checked to true will cause wrapping the original condition into **not()**. For example **not( $applicant : Applicant(age > 18) )**

**Comment by sterobin** [comment]
@jomarko, I corrected the Negate description. Thanks. Will cherry-pick this PR for 7.5.x.
@manstis, "Boolean" is a proper adj (George Boole) and thus generally capitalized. Also the called-for convention in tech comm style guides, including IBM, which Red Hat adheres to (mostly). If you know that it's "boolean" everywhere else in the UI, then I can make it lowercase. Let me know.

**Comment by manstis** [comment]
Thanks for the (```Boolean```) explanation @sterobin. 

Please let me know this turns green and I'll merge.

**Comment by sterobin** [comment]
@manstis , green for merge.

**Pull request merged**